src/RegExHelper.ts

```diff
@@ -189,6 +189,7 @@
 		result += this.getAttributesRegex(captureAttributes);
 		result += this.getModifiersRegex(captureModifiers);
 		result += this.getGenericTypeNameRegex(captureReturnType, false, false, false, false);
+		result += "[@]?";
```

Comment:
I think it should be added as part of the getGenericTypeNameRegex method. Also, I don't think the square brackets are needed? 🤔😊

Reply:
> I think it should be added as part of the getGenericTypeNameRegex method. Also, I don't think the square brackets are needed? 🤔😊

Does that mean '@' should be considered as a part of typename instead of property name?